plugins/parsers/collectd/parser.go

```diff
@@ -10,70 +10,63 @@ import (
 
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/metric"
+	"github.com/influxdata/telegraf/plugins/parsers"
 )
 
 const (
 	DefaultAuthFile = "/etc/collectd/auth_file"
 )
 
-type CollectdParser struct {
-	// DefaultTags will be added to every parsed metric
-	DefaultTags map[string]string
+type Parser struct {
+	MetricName  string            `toml:"metric_name"`
```

Comment:
I think we don't need to export this setting, do we?

Reply:
Probably not, I only added it because the other ones were doing it. But it originally wasn't there so not exporting it makes sense.